deepchem/feat/graph_features.py

```diff
@@ -274,20 +277,60 @@ def __init__(self, master_atom=False, use_chirality=False):
       the molecule.  This technique is briefly discussed in
       Neural Message Passing for Quantum Chemistry
       https://arxiv.org/pdf/1704.01212.pdf
-
+    use_chirality: Boolean
+      if true then make the resulting atom features aware of the 
+      chirality of the molecules in question
+    atom_properties: list of string or None
+      properties in the RDKit Mol object to use as additional
+      atom-level features in the larger molecular feature.  If None,
+      then no atom-level properties are used.  Properties should be in the 
+      RDKit mol object should be in the form 
+      atom XXXXXXXX NAME
+      where XXXXXXXX is a zero-padded 8 digit number coresponding to the
+      zero-indexed atom index of each atom and NAME is the name of the property
+      provided in atom_properties.  So "atom 00000000 sasa" would be the
+      name of the molecule level property in mol where the solvent 
+      accessible surface area of atom 0 would be stored. 
 
     Since ConvMol is an object and not a numpy array, need to set dtype to
     object.
     """
     self.dtype = object
     self.master_atom = master_atom
     self.use_chirality = use_chirality
+    self.atom_properties = atom_properties
+
+  def get_atom_properties(self, atom):
+    """
+    For a given input RDKit atom return the values of the properties
+    requested when initializing the featurize.  See the __init__ of the
+    class for a full description of the names of the properties
+    
+    Parameters
+    ----------
+    atom: RDKit.rdchem.Atom
+      Atom to get the properties of
+    returns a numpy lists of floats of the same size as self.atom_properties
+    """
+    values = []
```

Comment:
short circuit for none case of self.atom_properties.  This is also the reason you are failing all of the tests.

Reply:
defaulting to empty list is better than defaulting to None -- then you won't need the short circuit